calculate_movement_method.py

```diff
@@ -36,6 +39,13 @@ def lambda_handler(event, context):
     final_output = {}
 
     try:
+        # TESTING
```

Comment:
What are you testing? Why is this still here going into a PR?

Reply:
Resolved.